### ISSUE TITLE
Not able to accept developer agreement, getting bad gateway error

### ISSUE THREAD

**Issue by harryi3t** [canonical-websites/snapcraft.io#1383]
### Expected behavior
I should be able to verify my account and accept the developer agreement

### Steps to reproduce the problem

<img width="1131" alt="developer_program_agreement_ _linux_software_in_the_snap_store" src="https://user-images.githubusercontent.com/5207331/49289643-68958280-f4ca-11e8-92a1-1f29afc5d20a.png">

Clicking the continue button takes me to this page
<img width="765" alt="bad_gateway" src="https://user-images.githubusercontent.com/5207331/49289619-5582b280-f4ca-11e8-9f9b-7b35e1fdaa62.png">

Clicking the "status.snapcraft.io" link takes me to this page. Looks like the query parameter is causing issues.
<img width="875" alt="https___status_snapcraft_io___ga_2_59851821_1669471605_1543580776-288101521_1543299779" src="https://user-images.githubusercontent.com/5207331/49289681-8bc03200-f4ca-11e8-908e-9f6b0e5595ea.png">

Removing the query parameter makes the status page load correctly. But I do not see any incident for the bad gateway error.
<img width="464" alt="snap_store_status" src="https://user-images.githubusercontent.com/5207331/49289700-9ed30200-f4ca-11e8-9b0f-a46f01bb0425.png">

I tried using the old dashboard page https://dashboard.snapcraft.io/ as suggested by @tbille [here](https://github.com/canonical-websites/snapcraft.io/issues/1040#issuecomment-418748058), but again I end up like this
<img width="835" alt="dashboard_snapcraft_io" src="https://user-images.githubusercontent.com/5207331/49290009-acd55280-f4cb-11e8-986f-3cffd534f9d4.png">

Tried this on two different browsers on two different machines, same error.

**Comment by tbille** [comment]
@harryi3t thank you for reporting this bug!

Could you try to logout from https://login.ubuntu.com and try to login again via https://dashboard.snapcraft.io ?
We are trying to investigate where could it comes from :smile:

**Comment by cprov** [comment]
@harryi3t from our investigation it is likely that your browser is holding old/deprecated cookies for *.ubuntu.com and, consequently, *.snapcraft.io domains.
Can you please clear cookies for those domains before you try to log in again ?

**Comment by harryi3t** [comment]
@tbille @cprov 
I could swear that I tried that before (clearing cookies and even running in incognito mode) and did not work.
I thought it happened maybe because I used a separate system to click on the validation link sent to my email.
So, I had created another account and that worked.
Today when I tried that old account again in the incognito mode, it worked!

Thank you for looking into this issue. I am closing this ticket since the issue is resolved.
